pipelines/README.md

```diff
@@ -1,330 +1,37 @@
-  to disk. This is not used if there are less than 100 records. Use `0` to use
-  the default Parquet row-group size. Default: `0`.
-
-### Common questions
-
-- **Do I need to have a local HAPI server to load to?** No, this is only needed
-  if you want to test sending data to a local HAP FHIR server and is not needed
-  if you are only generating Parquet files.
-
-- **Will this pipeline include historical data recorded before the
-  `mysql binlog` was enabled?** Yes. By default, the pipeline takes a snapshot
-  of the entire database and will include all historical data.
-
-- **How do I stop Debezium from taking a snapshot of the entire database?** Set
-  `snapshotMode` in the config file to `schema_only` i.e,
-  `"snapshotMode" : "initial"`. Other options include: `when_needed`,
-  `schema_only`, `initial` (default), `never`, e.t.c. See the
-  [`debezium documentation`](https://camel.apache.org/components/latest/debezium-mysql-component.html)
-  for more details.
-
-- **What do I do if running `docker-compose` to stand up the OpenMRS image
-  fails** If `docker-compose` fails, you may need to adjust file permissions. In
-  particular if the permissions on `mysqld.cnf` is not right, the `datadir` set
-  in this file will not be read by MySQL, and it will cause OpenMRS to require
-  its `initialsetup` (which is not needed since the MySQL image already has all
-  the data and tables needed):
-
-  ```shell
-  $ docker-compose -f docker/openmrs-compose.yaml down -v
-  $ chmod a+r docker/mysql-build/mysqld.cnf
-  $ chmod -R a+r ./utils
-  $ docker-compose -f docker/openmrs-compose.yaml up
-  ```
-
-- **How do I see the demo data in OpenMRS?** In order to see the demo data in
-  OpenMRS you must rebuild the search index. In OpenMRS go to **Home > System
-  Administration > Advanced Administration**. Under **Maintenance** go to
-  **Search Index** then **Rebuild Search Index**.
-
-## How to query the data warehouse
-
-See [`dwh` directory](../dwh).
+# FHIR Data Pipes
+
+FHIR Data Pipes provides pipelines for transforming data from a FHIR server into
+a queryable format that you can use for data analysis and visualization. It can
+also be used to move data from one FHIR server to another. FHIR Data Pipes is
+built on technology designed for horizontal scalability and has multiple
+deployment options depending on your use case.
+
+There are a few ways to run FHIR Data Pipes; see these pages to learn more about
+them:
+
+- [FHIR Data Pipes Pipelines](https://github.com/google/fhir-data-pipes/tree/master/pipelines)
+  is a Java JAR which moves and transforms data from a FHIR server to Apache
+  Parquet files for data analysis or another FHIR server for data integration.
+  Pipelines is built using Apache Beam and can run on a single machine or
+  cluster.
+
+- [The Pipelines Controller module](https://github.com/google/fhir-data-pipes/tree/master/pipelines/controller)
+  transforms data from a HAPI FHIR server to an Apache Parquet data warehouse.
```

Comment:
What is the "merger" pipeline? I don't think it's documented anywhere.

Reply:
The way the incremental pipeline works is by fetching the resources that have been modified since last run, then "merge" these resources with the old ones. [This is the actual pipeline](https://github.com/google/fhir-data-pipes/blob/master/pipelines/batch/src/main/java/org/openmrs/analytics/ParquetMerger.java). The reason it is needed is because we need to consolidate duplicated resources (old resources that have been updated). I am not sure if we want to document it too much (you can assume it is implementation details) but if you feel mentioning it here can be confusing, please feel free to remove "merger".

Reply:
Added your change for now, but I'm not sure there's an end-user meaningful difference between merger and incremental, so maybe worth revisiting.